packages/nocodb/src/plugins/smtp/SMTP.ts

```diff
@@ -23,13 +24,11 @@ export default class SMTP implements IEmailAdapter {
         typeof this.input?.ignoreTLS === 'boolean'
           ? this.input?.ignoreTLS
           : this.input?.ignoreTLS === 'true',
-      auth: {
-        user: this.input?.username,
-        pass: this.input?.password,
-      },
       tls: {
         rejectUnauthorized: this.input?.rejectUnauthorized,
       },
+      ...(this.input?.username ? { auth: { user: this.input?.username } } : {}),
+      ...(this.input?.password ? { auth: { pass: this.input?.password } } : {}),
```

Comment:
this is wrong if both are defined password will overwrite the other.

Should be something like:

```typescript
...(this.input?.username || this.input?.password ? { auth: { 
      ...(this.input?.username ? {user: this.input?.username } : {}),
      ...(this.input?.password ? { pass: this.input?.password } : {}),
})
```

Reply:
Missed them. Fixed in  [be61a25](https://github.com/nocodb/nocodb/pull/8412/commits/be61a25adfcd72fcc98b065d8056a8cceba75542)